cookbook/custom-urls.rst

```diff
@@ -0,0 +1,82 @@
+How to manage custom-urls?
+==========================
+
+Sulu gives the content-manger an easy way to manage custom-urls or landing
+pages. This urls can be defined in the webspace-settings. A custom-url is
+basically an additional url for a specific page.
```

Comment:
I would also add somehow that the localization is already included (without the localization specific part int he URL)